externals/CorrelationIdGenerator.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
```

Comment:
For my information, any idea why some bits of ASP.NET Core are Apache, and not MIT?